backend/crud/notification.py

```
@@ -1,7 +1,8 @@
-from sqlalchemy.orm import Session
+from sqlalchemy.orm import Session, joinedload
 from models.notifications import Notification
 from schemas.notification import NotificationCreate
```

Ruff (0.8.2)

`schemas.notification.NotificationCreate` imported but unused

Remove unused import: `schemas.notification.NotificationCreate`

(F401)